runtime/seq_test.go

```diff
@@ -43,6 +43,43 @@ func TestSeqApply(t *testing.T) {
 	}
 }
 
+func TestSeqCount(t *testing.T) {
+	fun := wrapFuncForTest(func(f *Frame, seq *Object, v *Object) (*Object, *BaseException) {
```

Comment:
You could wrap seqCount directly instead of creating a new function: wrapFuncForTest(seqCount)